File: for/main02.c

//编一个程序，求出1到100之间的奇数之和

#include <stdio.h>
int main(int argc, const char * argv[]) {
    int sum = 0;
    for(int i = 1; i < 100; i++)
        if(i % 2 != 0) sum += i;
    printf("%d \n",sum);
}

File: 宏定义/main02.m

//定义一个带参的宏，实现两个整数之间的交换

#import <Foundation/Foundation.h>
#include <stdio.h>
#define CG(a,b,temp) temp = a; a = b; b = temp

int main(int argc, const char * argv[])
{
    int a = 5,b = 6,temp = 0;
    CG(a,b,temp);
//    temp = a;
//    a = b;
//    b = temp;
    printf("%d %d \n",a,b);
    return 0;
}
